src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -5003,3 +5003,5 @@ OBA:2044917	level of serine-rich single-pass membrane protein 1 (human) in blood
 OBA:2044918	level of uncharacterized protein C22orf15 (human) in blood serum	PR:Q8WYQ4	uncharacterized protein C22orf15 (human)	PATO:0000070	amount	UBERON:0001977	blood serum
 OBA:2044919	level of small integral membrane protein 10 (human) in blood serum	PR:Q96HG1	small integral membrane protein 10 (human)	PATO:0000070	amount	UBERON:0001977	blood serum
 OBA:2044920	level of uncharacterized protein C20orf173 (human) in blood serum	PR:Q96LM9	uncharacterized protein C20orf173 (human)	PATO:0000070	amount	UBERON:0001977	blood serum
+OBA:2020006	level of (5z7e)-(3s)-262626-trifluoro-27-nor-910-seco-5710(19)-cholestatriene-325-diol in blood serum	LM:LMST03020024	(5z7e)-(3s)-262626-trifluoro-27-nor-910-seco-5710(19)-cholestatriene-325-diol	PATO:0000070	amount	UBERON:0001977	blood serum
+OBA:2020007	level of triacylglycerol (56:6) in blood serum	SLM:000308352	triacylglycerol (56:6)	PATO:0000070	amount	UBERON:0001977	blood serum
```

Comment:
@rays22 I have added two examples here and tested them, I will leave the rest to you!